src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 4;
+    private static final int BOUND_RANDOM = 100;
+    private static final int CIRCLE_RADIUS = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private final Random randomNumber = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(CIRCLE_RADIUS, DEFAULT_COLOR);
+    }
+    
+    public Figure getRandomFigure() {
+        int indexFigure = randomNumber.nextInt(FIGURES_COUNT);
+        if (indexFigure == 0) {
+            return getRandomSquare();
+        } else if (indexFigure == 2) {
+            return getRandomRectangle();
+        } else if (indexFigure == 3) {
+            return getRandomRightTriangle();
+        } else {
+            return getIsoscelesTrapezoid();
+        }
```

Comment:
It's better to use `switch case` construction instead of tons `if else` statements
Pay attention:
1) You skip index 1, maybe it must be a `Circle`
2) You retrun **isoscelesTrapezoid** for all indexes that bigger than 3. According to the task description, you need to have a method `getDefaultFigure()` that returns a **Circle**

Reply:
I used switch instead if else, make sense. For calling the getDefaultFigure() method I called it in the class FigureCall (row 13). It's may implementation. Can it be like that? (the task couldn't clear describe it).